frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseSyncToParentByRebaseAction.java

```diff
@@ -189,14 +191,14 @@ private void doRebase(
       @UIThreadUnsafe
       public void run(ProgressIndicator indicator) {
 
-        AtomicReference<Try<Option<IExecutionResult>>> wrapper = new AtomicReference<>(Try.success(Option.none()));
+        final AtomicReference<Try<Option<IExecutionResult>>> wrapper = new AtomicReference<>(Try.success(Option.none()));
```

Comment:
The Nullness checker doesn't allow passing `null` to `Try.success()` instead of `Option.none`.
when changing the type to val, the `Option<IExecutionResult>` type cannot be inferred, due to initialisation with `Option.none()`

Reply:
One more thing to check (unless you've done so already):
```suggestion
        final AtomicReference<Try<@Nullable IExecutionResult>> wrapper = new AtomicReference<>(Try.success(Option.none()));
```